pack.go

```diff
@@ -126,6 +128,11 @@ var mediaTypeRegexp = regexp.MustCompile(`^[A-Za-z0-9][A-Za-z0-9!#$&-^_.+]{0,126
 //
 // artifactType and opts.ConfigDescriptor.MediaType MUST comply with RFC 6838.
 //
+// Each time when PackManifest is called, it generates a new time stamp in the
+// manifest annotations with the key "org.opencontainers.image.created". To make
```

Comment:
We also need to mention that if "org.opencontainers.image.created" is specified, it must conform to [RFC 3339](https://www.rfc-editor.org/rfc/rfc3339.html).

Reply:
Added a comment.